Assets/Tests/PlayMode/Extensions/ListExtensionsTests.cs

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace PlayEveryWare.EpicOnlineServices.Tests.Extensions
+{
+    using NUnit.Framework;
+    using PlayEveryWare.EpicOnlineServices.Extensions;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    [TestFixture]
+    public class ListExtensionsTests
+    {
+        /// <summary>
+        /// Test to verify that the Shuffle method changes the order of elements in the list.
+        /// </summary>
+        [Test]
+        public void Shuffle_ChangesOrderOfElements()
+        {
+            // Arrange
+            List<int> originalList = Enumerable.Range(1, 100).ToList();
+            List<int> shuffledList = new(originalList);
+
+            // Act
+            shuffledList.Shuffle();
```

Comment:
funny nit: This could *theoretically* result in a list that shuffles itself perfectly back in to place. HOWEVER, Wolfram Alpha says this is "1 in 100!", which means `1` in `93326215443944152681699238856266700490715968264381621468592963895217599993229915608941463976156518286253697920827223758251185210916864000000000000000000000000` chance of that happening.